extensions/wikia/AdEngine/js/provider/remnantGpt.js

```diff
@@ -27,6 +28,10 @@ define('ext.wikia.adEngine.provider.remnantGpt', [
 		{
 			afterSuccess: function (slotName) {
 				slotTweaker.removeDefaultHeight(slotName);
+				if (!uapContext.isBfaaLoaded()) {
```

Comment:
Something went wrong with resolving conflicts. This code should be here.

Reply:
done